client/src/containers/projects/form/utils.ts

```diff
@@ -42,8 +43,11 @@ export const parseFormValues = (data: CustomProjectForm) => {
       ecosystem: data.ecosystem,
       activity: data.activity,
       countryCode: data.countryCode,
-      // @ts-expect-error fix later
-      restorationActivity: data.parameters?.restorationActivity,
+      ...(data.activity &&
+        ACTIVITY.RESTORATION && {
```

Comment:
ACTIVITY.RESTORATION always evaluates to true. Or am I missing something here?

Reply:
You are right, sir.